start_bundle_coupled.R

```diff
@@ -613,8 +618,6 @@ for (scen in common) {
 }
 
 if (! "--test" %in% flags && ! "--gamscompile" %in% flags) {
-  system(paste("cp", file.path(path_remind, ".Rprofile "), file.path(path_magpie, ".Rprofile")))
```

Comment:
Should be kept but only executed if `is.null(renv::project())` in order to keep snapshots running.